languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -83369,6 +83369,48 @@ The accident victim died from her injuries.
                 <example>He <marker>is baking</marker> a cake.</example>
             </rule>
         </rulegroup> <!-- end passive voice rule -->
+		<rule id="OCCASION_TRANSITIVE_VERB_ARCHAIC" name="transitive verb 'occasion' is archaic" default="temp_off">
+			<antipattern>
+				<token postag="N.*" postag_regexp="yes" inflected="yes">occasion</token>
+			</antipattern>
+			<!-- Avoids false positives caused by the VBZNNS_NNS disambiguator incorrectly returning
+			only VBZ for 'occasions'. See the set of examples marked (*) below. -->
+			<antipattern>
+				<token>on</token>
+				<token postag="JJ|CD|RB" postag_regexp="yes" min="0" max="3" />
+				<token inflected="yes">occasion</token>
+			</antipattern>
+			<pattern>
+				<marker><token postag="VB.*" postag_regexp="yes" inflected="yes">occasion</token></marker>
+			</pattern>
+			<message>The transitive verb "<match no="1" />" is an archaic word choice. Consider a more common synonym, such as <!--
+				--><suggestion><match no="1" postag="VB.*">cause</match></suggestion>, <!--
+				--><suggestion><match no="1" postag="VB.*">induce</match></suggestion>, or <!--
+				--><suggestion><match no="1" postag="VB.*">bring</match> about</suggestion>
+			</message>
+			<url>https://tinyurl.com/yyxj28z8</url>
```

Comment:
This is interesting, but we should use the `url` to point to explanations, not to pure data. Also, it's okay to use long URLs, no need for tinyurl.